src/tests/test_protocol.py

```diff
@@ -4,17 +4,14 @@
 
 import unittest
 
-from .common import skip_python3
-
-skip_python3()
-
 
 class TestProtocol(unittest.TestCase):
     """Main protocol test case"""
 
     def test_check_local(self):
         """Check the logic of TCPConnection.local"""
         from pybitmessage import protocol, state
+        # import pdb; pdb.set_trace()
```

Comment:
this shouldn't be in a PR, remove before making a PR.

Reply:
Done